crates/proof-of-sql-sdk/examples/produce-plan/main.rs

```diff
@@ -0,0 +1,26 @@
+mod args;
+use crate::args::ProducePlanArgs;
+use clap::Parser;
+use proof_of_sql::{base::try_standard_binary_serialization, sql::evm_proof_plan::EVMProofPlan};
+use sxt_proof_of_sql_sdk::produce_plan;
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn core::error::Error>> {
+    // Parse command-line arguments
+    let args = ProducePlanArgs::parse();
+
+    // Retrieve the proof plan
+    let plan = produce_plan(args.substrate_node_url, &args.query, args.block_hash).await?;
+
+    if args.display_as_unserialized_plan {
+        println!("{:?}", plan);
+    } else if args.is_evm_plan {
+        let serialized = hex::encode(try_standard_binary_serialization(EVMProofPlan::new(plan))?);
+        println!("{:?}", serialized);
```

Comment:
The output for these is the hex bytes in utf8, but the debug formatting for strings puts quotes around it. I think it'd be the best UX to not have quotes and to have it prepended w/ `0x` (pretty common for our APIs to recognize `0x`-prepends as hex-encoded byte blobs)

```suggestion
        println!("0x{}", serialized);
```